src/PaymentPart/Output/OutputInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Sprain\SwissQrBill\PaymentPart\Output;
+
+use Sprain\SwissQrBill\QrBill;
+
+interface OutputInterface
+{
+    public function getQrBill(): ?QrBill;
+
+    public function getLanguage(): ?string;
+
+    public function getPaymentPart();
+
+    public function setPrintable(bool $printable);
```

Comment:
Why is it needed in an interface?

Reply:
Every future output should support two versions: pdf only and printable. This is how it should be set in all outputs.